src/ConfigSetup.cpp

```diff
@@ -313,21 +334,25 @@ void ConfigSetup::Init(const char *fileName, MultiSim const*const& multisim)
       in.restart.restartFromXSCFile = true;
     } else if(CheckString(line[0], "Checkpoint")) {
       if (line.size() == 3){
-        in.files.checkpoint.defined[0] = checkBool(line[1]);
-        in.restart.restartFromCheckpoint = checkBool(line[1]);
+        in.files.checkpoint.defined[0] = in.restart.restartFromCheckpoint = checkBool(line[1]);
```

Comment:
We also need to consider the cases where user put comments in the config file. Like:
```
Checkpoint   false ! specifies the checkpoint 
```

Reply:
We can use this method instead:
```
in.files.checkpoint.defined[0] = true;
if (line.size() >= 3) {
  in.restart.restartFromCheckpoint = checkBool(line[1]);
  if (multisim != NULL) {
    in.files.checkpoint.name[0] = multisim->replicaInputDirectoryPath + line[2];
  } else {
    in.files.checkpoint.name[0] = line[2];
  }
} else if (line.size() >= 2) {
  in.restart.restartFromCheckpoint = checkBool(line[1]);
}
```

Reply:
That will terminate with an error if the user enters

Checkpoint filename

which we currently allow with an assumption of true.

It is a good idea to handle comments, but I don't want to do it in this pull request.